test/integration/options/posixExitCodes.spec.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+var helpers = require('../helpers');
+var runMocha = helpers.runMocha;
+
+describe('--posix-exit-codes', function () {
+  describe('when enabled, and with node options', function () {
+    var args = ['--no-warnings', '--posix-exit-codes'];
+
+    it('should exit with code 134 on SIGABRT', function (done) {
+      var fixture = 'signals-sigabrt.fixture.js';
+      runMocha(fixture, args, function postmortem(err, res) {
+        if (err) {
+          return done(err);
+        }
+        expect(res.code, 'to be', 134);
+        done();
+      });
+    });
+
+    it('should exit with code 143 on SIGTERM', function (done) {
+      var fixture = 'signals-sigterm.fixture.js';
+      runMocha(fixture, args, function postmortem(err, res) {
+        if (err) {
+          return done(err);
+        }
+        expect(res.code, 'to be', 143);
+        done();
+      });
+    });
+
+    it('should exit with code 0 even if there are test failures', function (done) {
```

Comment:
I think it's a good idea. 👍

Reply:
it('should exit normally with EXIT_SUCCESS',

Reply:
I think the `normal` vs `abnormal` nomenclature was dropped from the posix nomenclature [See: https://pubs.opengroup.org/onlinepubs/009695299/functions/exit.html].

If we go with @JoshuaKGoldberg 's suggestion here, it would be `it(should exit with EXIT_FAILURE)`.  That lines up with what I was able to find on the standard use of 0 vs 1 as exit codes[*]:

  * 0: Success—Indicates that the command or program executed successfully without any errors.
  * 1: General Error—A catch-all exit code for a variety of general errors. Often used when the command or program encounters an error, but no specific exit code is available for the situation.